src/main/java/io/github/opencubicchunks/cubicchunks/world/level/cube/LevelCube.java

```diff
@@ -420,39 +393,29 @@ public void unregisterTickContainerFromLevel(ServerLevel level) {}
     public native void registerAllBlockEntitiesAfterLevelLoad();
 
     // TODO (P3): GameEvent stuff is a bit concerning
-    @TransformFromMethod(
-        value = @MethodSig("addGameEventListener(Lnet/minecraft/world/level/block/entity/BlockEntity;Lnet/minecraft/server/level/ServerLevel;)V"),
-        owner = @Ref(LevelChunk.class)
-    )
+    @TransformFromMethod(value = @MethodSig("addGameEventListener(Lnet/minecraft/world/level/block/entity/BlockEntity;Lnet/minecraft/server/level/ServerLevel;)V"), owner = @Ref(LevelChunk.class))
     private native <T extends BlockEntity> void addGameEventListener(T blockEntity, ServerLevel level);
 
     @TransformFromMethod(value = @MethodSig("updateBlockEntityTicker(Lnet/minecraft/world/level/block/entity/BlockEntity;)V"), owner = @Ref(LevelChunk.class))
     private native <T extends BlockEntity> void updateBlockEntityTicker(T blockEntity);
 
-    @TransformFromMethod(
-        value = @MethodSig("createTicker(Lnet/minecraft/world/level/block/entity/BlockEntity;Lnet/minecraft/world/level/block/entity/BlockEntityTicker;)"
-            + "Lnet/minecraft/world/level/block/entity/TickingBlockEntity;"),
-        owner = @Ref(LevelChunk.class)
-    )
+    @TransformFromMethod(value = @MethodSig("createTicker(Lnet/minecraft/world/level/block/entity/BlockEntity;Lnet/minecraft/world/level/block/entity/BlockEntityTicker;)"
+            + "Lnet/minecraft/world/level/block/entity/TickingBlockEntity;"), owner = @Ref(LevelChunk.class))
     private native <T extends BlockEntity> TickingBlockEntity createTicker(T blockEntity, BlockEntityTicker<T> ticker);
 
     // TODO forge stuff
     // FORGE START
 //    private final net.neoforged.neoforge.attachment.AttachmentHolder.AsField attachmentHolder = new net.neoforged.neoforge.attachment.AttachmentHolder.AsField();
 //
-//    @Override
-//    public boolean hasData(net.neoforged.neoforge.attachment.AttachmentType<?> type) {
+//    @Override //    public boolean hasData(net.neoforged.neoforge.attachment.AttachmentType<?> type) {
```

Comment:
:laughing:

Reply:
bit goofy, but it only happens since the code was commented out before formatting was applied